### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by foresmac**
- added a summary of configuration options
- added docs for @WorkHorseIndustries’s pre-warming header option
- preparing for blog post

@vokal/systems

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,35 +7,23 @@
...
-
-Resized images may also be center-cropped by passing `?c=true` in the querystring:
-        
-        images.example.com/mybucket/5272a0e7d0d9813e21?s=160&c=true
+When images are requested they are placed into an in-memory cache to make repeated requests for that image faster.
 
-The thumbnail will then be cached to both `groupcache` and S3. If the image leaves
-the in-memory cache it will not need to be resized again.
+You can resize an image on the fly by providing an `?s=X` parameter that specifies
+a maximum width for the image. For example, if you want to resize an image down to a 160 pixel thumbnail: `http://images.example.com/mybucket/5272a0e7d0d9813e21?s=160`. Resized images may also be center-cropped by passing `?c=true` in the querystring: `images.example.com/mybucket/5272a0e7d0d9813e21?s=160&c=true`. The thumbnail will then be cached to both `groupcache` and S3. If the image leaves the in-memory cache it will not need to be resized again.
```
Comment (vhoof) [inline README.md]: IMO, the above paragraph looked a little better with the URLs on their own lines. In the new one, the urls are big code blocks that cause line-length weirdness anyway, so breaking them out ends up clearer and less funky.

File: README.md [hunk 2]
```diff
@@ -70,6 +68,24 @@ $ docker run -e AUTH_TOKEN=... -e AWS_SECRET_ACCESS_KEY=... \
...
+- `AUTH_TOKEN`: A secret token that non-browser clients can use to autheticate for uploads. If no token is supplied, anyone could upload to your image proxy
+- `ALLOWED_ORIGIN`: a comma-delimited list of hostnames to accept CORS requests from browser-based clients, e.g. `www.example.com,*.example2.com` will accept uplaod requests from pages originating from www.example.com or any subdomain of example2.com. If this is not set, CORS is disabled and will likely fail for any upload requests from a browser.
+- `VIP_SIZE_LIMIT`: A maximum file-size limit in megabytes (default `5`)
+- `VIP_MAX_WIDTH`: A maximum width for resized images in pixels (default `720`)
+
+For serving via HTTPS (recommended), `vip` expects to find an SSL certificate as well as the matching private key in the following locations:
+- `/etc/vip/application.pem`
+- `/etc/vip/application.key`
+
+By default, `vip` will also response to HTTP/2 requests; however, some mobile clients have incomplete implementations and may fail. You can turn off HTTP/2 support by setting `DISABLE_HTTP2` to `True`.
```
Comment (vhoof) [inline README.md]: s/response/respond, presumably.

**Comment by vhoof** [comment]
LGTM.

**Comment by foresmac** [comment]
Thanks @vhoof

**Pull request merged**